sdxlib/sdx_client.py

```diff
@@ -517,7 +547,13 @@ def create_l2vpn(self) -> SDXResponse:
             return SDXResponse(response_json)
 
         try:
-            response = requests.post(url, json=payload, timeout=120)
+            # Include authentication if username and password are provided.
+            auth = (
+                (self._http_username, self._http_password)
+                if self._http_username and self._http_password
+                else None
+            )
+            response = requests.post(url, json=payload, auth=auth, timeout=120)
```

Comment:
Hi @SColson82 requests lib internally handle authentication when the username and password are none, you can simplify the code above with just:
```suggestion
            # Include authentication if username and password are provided.
            auth = (self.http_username, self.http_password)
            response = requests.post(url, json=payload, auth=auth, timeout=120)
```

Reply:
Hi @italovalcy! Thank you for this feedback, these changes have been implemented and the unit tests have been adjusted to account for them.